Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -1175,4 +1186,17 @@ public int getThemeId()
   {
     return mThemeId;
   }
+
+  private static boolean isVRInstalled(Context context)
+  {
+    PackageManager pm = context.getPackageManager();
+    for (ApplicationInfo app : pm.getInstalledApplications(PackageManager.GET_META_DATA))
```

Comment:
Don't you need to declare the VR package name in the manifest so we're allowed to query for it? https://developer.android.com/training/package-visibility/declaring

Reply:
Have you tested that on Android 11 or higher?

Reply:
You were right about this. I just fixed it.